docs/index.md

```diff
@@ -1,9 +1,22 @@
 # Helm: The Kubernetes Package Manager
 
-Helm helps you bootstrap your Kubernetes cluster with **Charts** that provide ready-to-use workloads like:
+Helm helps you find and use software built for Kubernetes. The Helm CLI uses Charts which not only contain metadata about software packages but also Pod, ReplicationController and Service manifests for Kubernetes. With a few Helm commands you can quickly and easily deploy software packages like:
 
-- A Postgres database
-- A scale-out etcd cluster
-- An HAProxy Edge Load Balancer
+- Postgres
```

Comment:
I think it may be too distracting to the reader. These are fairly common services known by the majority of the software industry, so a link to introduce them would be pointless.